src/clients/rust/tests/tests.rs

```diff
@@ -0,0 +1,302 @@
+use anyhow::Result as AnyResult;
+use futures::executor::block_on;
+use std::process::{Child, Command};
+use std::sync::atomic::{AtomicU16, Ordering};
+use std::{env, fs};
+use tempfile::TempDir;
+
+use tigerbeetle as tb;
+
+const FIRST_PORT_NUMBER: u16 = 20_000;
+static NEXT_PORT_NUMBER: AtomicU16 = AtomicU16::new(FIRST_PORT_NUMBER);
+
+struct TestHarness {
+    tigerbeetle_bin: String,
+    temp_dir: TempDir,
+    port_number: u16,
+    server: Option<Child>,
+}
+
+impl TestHarness {
+    fn new(name: &str) -> AnyResult<TestHarness> {
```

Comment:
I usually spell this as `anyhow::Result<TestHarness>`, to avoid aliases, though no strong opinion here.

Reply:
Done.